### ISSUE TITLE
Automate meeting recording, transcripts and notes for your Google Meet meetings

### ISSUE THREAD

**Issue by jkaufman-mitre** [cisagov/ScubaGoggles#500]
Create a SHOULD policy stating that this feature should be disabled.

**Comment by jkaufman-mitre** [comment]
This is a Gemini feature, so will be placed on hold for now until Gemini SCB.

**Comment by mdueltgen** [comment]
Added in new policies for automated recording and transcription, waiting on TTP mappings before opening PR
